persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -82,18 +89,35 @@ class HmdaFileValidator(submissionId: SubmissionId) extends HmdaPersistentActor
 
   import HmdaFileValidator._
 
+  var institution: Option[Institution] = None
+
   val config = ConfigFactory.load()
   val duration = config.getInt("hmda.actor-lookup-timeout")
 
   implicit val timeout = Timeout(duration.seconds)
 
   val parserPersistenceId = s"${HmdaFileParser.name}-$submissionId"
 
+  def ctx: ValidationContext = ValidationContext(institution, Try(Some(submissionId.period.toInt)).getOrElse(None))
+
   var state = HmdaFileValidationState()
 
   val supervisor = system.actorSelection("/user/supervisor")
   val fHmdaFiling = (supervisor ? FindHmdaFiling(submissionId.period)).mapTo[ActorRef]
 
+  val submissionLarStats = context.actorOf(SubmissionLarStats.props(submissionId))
+
+  override def preStart(): Unit = {
```

Comment:
We also need to use `ctx` in the `BeginValidation` method. Currently is uses the one defined on line 140, which doesn't supply the institution.